doc/news/_preparation_next_release.md

```diff
@@ -244,7 +244,7 @@ This section keeps you up-to-date with the multi-language support provided by El
 
 ## Documentation
 
-- <<TODO>>
+- Adapt and remove outdated docs https://github.com/ElektraInitiative/libelektra/issues/4882 _(Tomislav Makar @tmakar)_
```

Comment:
```suggestion
- Adapt and remove outdated docs https://issues.libelektra.org/4882 _(Tomislav Makar @tmakar)_
```

Better use issues.libelektra.org instead of a direct GitHub link